web-common/src/main/java/edu/unc/lib/boxc/web/common/utils/AnalyticsTrackerUtil.java

```diff
@@ -45,8 +45,7 @@ public class AnalyticsTrackerUtil {
     protected static final String DEFAULT_CID = "35009a79-1a05-49d7-b876-2b884d0f825b";
     // Google analytics measurement API url
     private static final String GA_URL = "https://www.google-analytics.com/collect";
-    private static final int MATOMO_SITE_ID = 3;
-    private static final String MATOMO_ACTION = "download";
+    public static final String MATOMO_ACTION = "Downloaded";
```

Comment:
I updated it only because I thought it might be good to keep it in line with the other category/event that we have, which is Links / Clicked. I could update it to Downloaded Original, since it's a good point we might need to update it to other sizes in the future